Source/Core/CommandLineOptions.cs

```diff
@@ -1656,6 +1687,14 @@ protected override bool ParseOption(string name, CommandLineOptionEngine.Command
         case "kInductionDepth":
           ps.GetNumericArgument(ref KInductionDepth);
           return true;
+        
+        case "emitDebugInformation":
+          ps.GetNumericArgument(ref emitDebugInformation);
+          return true;
+        
+        case "discardNames":
```

Comment:
I find the option name "discardNames" confusing.  I have indicated several times before in my feedback that the verb "discard" is inappropriate for what this PR is trying to achieve.

Reply:
Hey, sorry for not responding to your earlier comment.

Do you have a suggestion for a different name? Your earlier comment seems to indicate something with `rename`, but I find that `renameNames` does not say enough about what the option does.

The goal of the option is to get verification output that does not depend on original program names and is deterministic. I picked `discardNames` to indicate that the original program names are not used, leaving the reader to conclude that they must be replaced using some fixed naming scheme. 

Here are some ideas for other names:
`replaceNames`, `discardAndReplaceNames`, `ignoreNames`, or `renameBasedOnContent`

Do you have a preference or suggestion?

Reply:
Maybe `normalizeNames`?

Reply:
normalizeNames appears to be the best option so far.

Reply:
Great, made the change :)